app/views/shelters/_form.html.erb

```diff
@@ -63,6 +63,12 @@
     <%= f.label :source %>
     <%= f.text_field :source %>
 
+    <%= f.label :phone %>
+    <%= f.text_field :phone, :required => true %>
```

Comment:
Looks like you might have resolved a merge conflict incorrectly -- the phone field was added above on line 37.

Reply:
Thank you for your comment. Updated.